### ISSUE TITLE
[BUG] Schema::getColumnType, "Undefined index: COMMENTS"

### ISSUE THREAD

**Issue by deepak-a-insnapsys** [yajra/laravel-oci8#674]
### Summary of problem or feature request

On running the migration fresh command I'm getting this error 

```
Undefined index: COMMENTS

  at vendor/doctrine/dbal/lib/Doctrine/DBAL/Schema/OracleSchemaManager.php:414
```

### Code snippet of problem
![image](https://user-images.githubusercontent.com/71549629/134461546-394ccf8b-9e61-4d5d-9f16-e02948bbbef7.png)

### I've added a dump statement in the OracleSchemaManager.php where I'm getting the above error. 
here is the snippet of it
![image](https://user-images.githubusercontent.com/71549629/134462149-91dc5f1b-4cb2-49c4-a14b-06c7b8c5e303.png)

**and what I found is, in the $tableOptions array "comments" keyword is in the lowercase and In the OracleSchemaManger file its uses in the Uppercase** 

![image](https://user-images.githubusercontent.com/71549629/134462592-9eff57da-48c3-457b-a906-03f612cce639.png)

![image](https://user-images.githubusercontent.com/71549629/134462433-6d3ea864-b0ec-4224-8a96-06642261092e.png)

### System details

- Operating System => centos 7
- PHP Version => 7.3.27
- Laravel Version => 8.0
- Laravel-OCI8 Version => 8.3.0

@yajra

**Comment by yajra** [comment]
Can you provide some snippets to reproduce the issue? The package returns lowercase object names / keys. I think we need to find where the code is being triggered and maybe force an uppercase "COMMENTS" before calling any related functions.

**Comment by deepak-a-insnapsys** [comment]
@yajra 

To reproduce above issue install following two packages
- "yajra/laravel-oci8": "^8.5"
- "doctrine/dbal": "^3.1"

What I found is, "doctrine/dbal": "^3.1" package creates OracleSchemaManger.php file under /DBAL/Schema/ directory.
In that, there is one function named "listTableDetails"  where they used "COMMENTS" in uppercase that caused the above issue 

![image](https://user-images.githubusercontent.com/71549629/150728714-0184cc02-30dd-4a43-b533-50b7c2f41fb5.png)


This function (listTableDetails) get called whenever we are using **Schema::getColumnListing("table_name")** function for the Oracle Database.

I hope this information will be helpful to you.

**Comment by yajra** [comment]
I just tried a fresh Laravel 8 installation and seems to work fine:

![image](https://user-images.githubusercontent.com/2687997/151739830-95458bf2-4b7b-4886-9eb1-9f5d94de96ec.png)

fresh migrate
![image](https://user-images.githubusercontent.com/2687997/151740133-5216f1a5-d210-4827-84fd-b34bb97da33d.png)

You might have a unique migration that causes the issue. Can you please provide that snippet? Can't replicate.

**Comment by deepak-a-insnapsys** [comment]
@yajra 


`Schema::getColumnType($table, $column)`
where this code is used I'm getting above error for the Oracle Database.

Tested with following packages with 
"yajra/laravel-oci8": "^8.6"
"doctrine/dbal": "^3.1"

![image](https://user-images.githubusercontent.com/71549629/151921857-cf67c2d7-8735-40d4-9601-c461288f79b4.png)

![image](https://user-images.githubusercontent.com/71549629/151921951-0a162974-f9a8-48a7-ad1b-ca67e2d1cf8b.png)

**Comment by yajra** [comment]
Found the issue and I think it can only be fixed on `doctrine/dbal` side. Will try to submit a PR if it will pass.

**Comment by deepak-a-insnapsys** [comment]
@yajra Thank you :). Looking forward to it.

**Comment by yajra** [comment]
PR sent, let's hope it gets accepted.

**Comment by yajra** [comment]
PR was merged. This should be fixed on doctrine/dbal:3.3.2 🚀
